front/src/modules/comments/components/CommentThreadRelationPicker.tsx

```diff
@@ -22,16 +20,19 @@ import { flatMapAndSortEntityForSelectArrayOfArrayByName } from '@/ui/utils/flat
 import { getLogoUrlFromDomainName } from '@/utils/utils';
 import {
   CommentableType,
+  GetCommentThreadQuery,
   useSearchCompanyQuery,
   useSearchPeopleQuery,
 } from '~/generated/graphql';
 
 import { MultipleEntitySelect } from '../../relation-picker/components/MultipleEntitySelect';
 import { useHandleCheckableCommentThreadTargetChange } from '../hooks/useHandleCheckableCommentThreadTargetChange';
+import { CommentableEntity } from '../types/CommentableEntity';
 import { CommentableEntityForSelect } from '../types/CommentableEntityForSelect';
 
 type OwnProps = {
-  commentThread: CommentThreadForDrawer;
+  commentThread?: GetCommentThreadQuery['findManyCommentThreads'][0];
```

Comment:
We shouldn't use query types ; the CommentThreadRelationPicker does not have to know about queries. We shoud use something like CommentThread or even better: Pick<CommentThread, 'id' | 'commentThreadTargets '>